applications/SwimmingDEMApplication/python_scripts/variables_management.py

```diff
@@ -434,12 +434,15 @@ def ConstructListsOfVariablesForCoupling(self, parameters):
             self.coupling_dem_vars += [Kratos.REYNOLDS_NUMBER]
 
         if self.do_backward_coupling:
-            if parameters["coupling"]["backward_coupling"]["apply_time_filter_to_fluid_fraction_option"].GetBool():
-                self.time_filtered_vars += [Kratos.FLUID_FRACTION_FILTERED]
+            if parameters["coupling"]["backward_coupling"]["apply_time_filter_to_fluid_fraction_option"].GetBool() or True:
+                self.time_filtered_vars += [Kratos.FLUID_FRACTION]
```

Comment:
This looks like a leftover from debugging

Reply:
True. I just removed it.